evaluator/abstract_evaluator.py

```diff
@@ -66,6 +66,12 @@ def get_config_file_name(cls, config_evaluator_type=None):
         return f"{TENTACLES_PATH}/{TENTACLES_EVALUATOR_PATH}/{config_evaluator_type}/{EVALUATOR_CONFIG_FOLDER}" \
             f"/{cls.get_name() + CONFIG_FILE_EXT}"
 
+    @classmethod
+    def get_config_file_error_message(cls, error):
```

Comment:
Where this method is used ?

Reply:
Oh ok in https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/60

Reply:
Why this is not used at all evaluator config load ?

Reply:
Because no one coded it in evaluator config load yet :)